releases.go

```diff
@@ -230,6 +230,17 @@ func createFormation(db *gorm.DB, release *Release) error {
 	return nil
 }
 
+// GetCurrentFormations gets the current formations for an app
+func GetCurrentFormations(db *gorm.DB, app *App) (*Formation, error) {
```

Comment:
No need to use a pointer for `map` types, since they're already passed by reference:

```go
func GetCurrentFormations(db *gorm.DB, app *App) (Formation, error)
```